client/finality-grandpa/rpc/Cargo.toml

```diff
@@ -8,16 +8,29 @@ edition = "2018"
 license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 
 [dependencies]
+sc-rpc = { version = "2.0.0-rc5", path = "../../rpc" }
+sp-runtime = { version = "2.0.0-rc5", path = "../../../primitives/runtime" }
 sc-finality-grandpa = { version = "0.8.0-rc5", path = "../" }
 finality-grandpa = { version = "0.12.3", features = ["derive-codec"] }
 jsonrpc-core = "14.2.0"
 jsonrpc-core-client = "14.2.0"
 jsonrpc-derive = "14.2.1"
+jsonrpc-pubsub = "14.2.0"
 futures = { version = "0.3.4", features = ["compat"] }
 serde = { version = "1.0.105", features = ["derive"] }
 serde_json = "1.0.50"
 log = "0.4.8"
 derive_more = "0.99.2"
+parity-scale-codec = { version = "1.3.0", features = ["derive"] }
 
 [dev-dependencies]
+sc-block-builder = { version = "0.8.0-rc5", path = "../../block-builder" }
+sc-network-test = { version = "0.8.0-rc5", path = "../../network/test" }
+sc-rpc = { version = "2.0.0-rc5", path = "../../rpc", features = ["test-helpers"] }
+sp-blockchain = { version = "2.0.0-rc5", path = "../../../primitives/blockchain" }
+sp-consensus = { version = "0.8.0-rc5", path = "../../../primitives/consensus/common" }
 sp-core = { version = "2.0.0-rc5", path = "../../../primitives/core" }
+sp-finality-grandpa = { version = "2.0.0-rc5", path = "../../../primitives/finality-grandpa" }
+sp-keyring = { version = "2.0.0-rc5", path = "../../../primitives/keyring" }
+substrate-test-runtime-client = { version = "2.0.0-rc5",  path = "../../../test-utils/runtime/client" }
```

Comment:
Is all of these really needed? Doesn't `substrate-test-runtime-client` re-export some of this stuff?

Reply:
`substrate-test-client` re-exports `Ed25519Keyring`, but that seems like an even bigger dependency? But maybe it's more correct to swap `sp-keyring` for `substrate-test-client`? 

The `sp-finality-grandpa` dependency is for `localized_payload` which I'm not seeing being re-exported